sdks/python/apache_beam/examples/snippets/snippets.py

```diff
@@ -175,8 +177,9 @@ def _add_argparse_args(cls, parser):
        'Or to take arms against a sea of troubles, '])
    | beam.io.WriteToText(my_options.output))
 
-  p.run()
+  result = p.run()
```

Comment:
Updated most of them. I could not easily update some of these in which the pipeline creation was part of the snippet and it was immediately followed by applying transforms inside the snippet.